app/src/components/ViewEvent/ViewEventContainer.tsx

```diff
@@ -164,12 +168,14 @@ export const ViewEventContainer = () => {
           </>
         )}
         <h1 className={style.header}>Påmeldte</h1>
-        {participants && participants.length > 0 ? (
-          participants.map(p => (
-            <div key={serializeEmail(p.email)} className={style.text}>
-              {stringifyEmail(p.email)}
-            </div>
-          ))
+        {participants.length > 0 ? (
+          [...participants].map(p => {
```

Comment:
![image](https://user-images.githubusercontent.com/13903284/75021590-f32e1d00-5494-11ea-83a5-f43c30eb3ae8.png)

Reply:
[...list].map er meir legit enn man skulle tro, fordi spreadoperatoren virker for alle itererbare ting i javascript, mens .map finnes bare på arrayprototypen. Selvfølgelig dersom `list` faktisk er ein array ser det litt dumt ut, men det slik man itererer over feks. Map typen.

Reply:
Men det man kan lure på er om `length > 0` sjekken er nødvendig 🤔

Reply:
`participants` er `Array` her, så memet gjelder

Reply:
Hvordan våger du å betvile _ mitt_  meme